edisgo/tools/temporal_complexity_reduction.py

```diff
@@ -503,6 +706,33 @@ def get_most_critical_time_intervals(
         of buses that reach their maximum voltage deviation in a certain time interval
         at a voltage deviation of higher or equal to 0.2*0.95.
         Default: 0.95.
+    weight_by_costs : bool
+        Defines whether overloading and voltage issues should be weighted by estimated
+        grid expansion costs or not. This can be done in order to take into account that
+        some grid issues are more relevant, as reinforcing a certain line or feeder will
+        be more expensive than another one.
+
+        In case of voltage issues:
+        If True, the costs for each MV and LV feeder, as well as MV/LV station are
+        determined using the costs for earth work and new lines over the full length of
+        the feeder respectively for a new MV/LV station. In each time interval, the
+        estimated costs are only taken into account, in case there is a voltage issue
+        somewhere in the feeder.
+        The costs don't convey the actual costs but are an estimation, as
+        the real number of parallel lines needed is not determined and the whole feeder
+        length is used instead of the length over two-thirds of the feeder.
+        If False, the severity of each feeder's voltage issue is set to be the same.
```

Comment:
If there is no specific reason to do so, I would probably change it, so it is consistent with the overloading. Or why would we handle the voltage deviation differently?

Reply:
It was set to 1 before and I just left it as it was, but was also wondering about it. I changed it now so that the maximum voltage deviation in the feeder is used.